openapi/components/schemas/SubscriptionChange.yaml

```
@@ -81,13 +81,24 @@ properties:
                   format: double
                   description: Usage limit amount in the currency of the order.
                   minimum: 0.01
-        trialEndTime:
-          description: Free trial item date time.
+        servicePeriodStartTime:
+          description: Date and time when the item service period starts.
```

suggestion: to avoid [noun stack](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
          description: Date and time when the service period of the item starts.
```